docs/docs/how-tos/using-argo.md

```diff
@@ -33,6 +33,35 @@ UI, you can check that things are working by running `argo list`.
 
 ![Argo Workflows User Tab](/img/tutorials/argo_workflows_user_tab.png)
 
+## Jupyterflow-Override (Beta)
+
+New users of Argo Workflows are often frustrated because the Argo Workflow pods do not have access to the same conda environments and shared files as the Jupyterlab user pod by default. To help with this use case, Nebari comes with [Nebari Workflow Controller](https://github.com/nebari-dev/nebari-workflow-controller) which overrides a portion of the Workflow spec when the
+`jupyterflow-override` label is applied to a workflow. The Jupyterlab user pod's environment variables, home and shared directories, docker image, and more will be added to the Workflow. Users can then e.g. run a script that loads and saves from their home directory with a particular conda environment. This works whether the label is added to the Workflow in a kubernetes manifest, via Hera, the argo CLI, or via the Argo Server Web UI. However, this does require that a jupyter user pod be running when the workflow is submitted. The Workflow pod will have the same resources (cpu, memory) that the user pod has.
+
+```
+api: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  name: jupyterflow-override-example
+  namespace: dev  # replace with your deployment namespace if needed
+  labels:
+    jupyterflow-override: 'true'
+spec:
+  entrypoint: print-hello-world
+  templates:
+    - name: print-hello-world
+      container:
+        name: main
+        image: any-image:any-tag  # will be overridden with the jupyter user pod image
```

Comment:
This is a bug that it runs on the general node and not the user nodes.  I'll fix that.  

> Given that the kubespawner has the extra_container_config overrides, do you know if those are also carried forward?
It would not copy over extra_container_config at the moment, but I could make that improvement to nebari_workflow_controller

Reply:
Fixed that bug.  They launch on the user nodes now